index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile First Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="#" class="title">Bilal Ahmed</a> 
+      <img src="./img/threelines.png" class="menu" alt="menu bar">
+    </nav>
+    <div class="main">
+      <h1 class="intro1">Hey There.I’m Bilal Ahmed</h1>
+      <h2 class="intro2">I’am a Software Developer</h2>
+      <p class="discription">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>
+    </div>
+    <ul class="icons">
+      <li class="github"><a href="https://github.com/bilalrajput09" target="_blank"><img src="./img/github.png" alt="github"></a></li>
+      <li class="linkdin" ><a href="https://www.linkedin.com/in/bilal-ahmed-18b12019a/" target="_blank"><img src="./img/linkdin.png" alt="linkdin"></a></li>
+      <li class="victory"><a href="" target="_blank"><img src="./img/1.png" alt="victory"></a></li>
+      <li class="twitter"><a href="https://twitter.com/bilal_rajput09" target="_blank"><img src="./img/twitter.png" alt="twitter"></a></li>
+      <li class="medium"><a href="https://medium.com/@ranabilalrajput09" target="_blank"><img src="./img/medium.png" alt="medium"></a></li>
+    </ul>
```

Comment:
Please consider wrapping it in `section tag` instead of `header tag` as it's a separate section. Moreover, wrap the header in header tag rather than `nav tag`. The reason is as its considered to be in best practices for semantcis.